src/main/java/com/example/Twitter/service/GetPost.java

```diff
@@ -0,0 +1,32 @@
+package com.example.Twitter.service;
+
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class GetPost implements TwitterServiceInterface{
+    public List<String> getTimeLine(Twitter twitter) throws TwitterException {
+        List<String> a = new ArrayList<>();
+        List<Status> s=twitter.getHomeTimeline();
+        for(Status sa : s){
```

Comment:
always use meaningful variable names
could be something like - Status status : listOfStatus